manim/_config/utils.py

```diff
@@ -1304,6 +1311,12 @@ def tex_template_file(self, val: str) -> None:
             self._d["tex_template_file"] = val  # actually set the falsy value
             self._tex_template = TexTemplate()  # but don't use it
 
+    plugins = property(
+        lambda self: self._d["plugins"],
+        lambda self, val: self._d.__setitem__("plugins", val),
+        doc="List of Plugins to Enable.",
+    )
```

Comment:
This is a strange pattern to me, not only because you're defining a property (and its getter and setter) without decorators, but also because you're manually calling a dunder method (`__setitem__` in this case). Though uh, looking at the rest of the file, it seems this is common, even though not all properties follow this pattern? That's... interesting. I guess it'd be fine then to keep it like this for consistency, but if you were to change it then I'd suggest this:
```suggestion
    @property
    def plugins(self):
        """List of plugins to enable."""

        return self._d["plugins"]

    @plugins.setter
    def plugins(self, value):
        self._d["plugins"] = value
```